src/main/java/com/epam/controller/ToursController.java

```diff
@@ -40,7 +48,21 @@ public ModelAndView searchTours(@RequestParam String country, @RequestParam Stri
         try {
             LocalDate addStartDate = Validator.getDate(startDate, true);
             LocalDate addEndDate = Validator.getDate(endDate, true);
-            toursModel.addObject("list", toursOfferService.searchTours(null, addStartDate, addEndDate));
+            List<Hotel> myList = hotelService.getHotelsByCountry(country);
+            if(myList.size()==0&&!country.isEmpty()){
```

Comment:
It seems like business logic, please replace it to the service layer

Reply:
Done